docs/root/intro/arch_overview/upstream/upstream.rst

```diff
@@ -4,6 +4,7 @@ Upstream clusters
 .. toctree::
   :maxdepth: 2
 
+  aggregate_cluster
```

Comment:
Can you move this to after the `load_balancing/load_balancing` item? This makes for a more logical ordering on the LHS index in docs.

Reply:
done